styles.css

```diff
@@ -0,0 +1,270 @@
+body {
+	background-color: #181818;
+	color: white;
+	margin: 0px;
+	font-family: 'Roboto', sans-serif;
+	display: block;
+}
+
+#navbar {
+	background-color: #202020;
+	display: grid;
+	grid-template-columns: 0.75fr 1.5fr 0.75fr;
+	grid-template-rows: 1fr;
+	justify-items: stretch;
+}
+
+#hamburger-button {
+	display: flex;
+	align-items: center;
+	margin-left: 20px;
+}
+
+.fas {
+	font-size: 1.2rem;
+	cursor: pointer;
+	font-weight: 900;
+	-webkit-font-smoothing: antialiased;
+	display: inline-block;
+	font-style: normal;
+	font-variant: normal;
```

Comment:
font-variant and font-style are set to normal by default, so you don't need to specify them, hence you can remove them.